README.md

```diff
@@ -145,9 +145,11 @@ This section provides a high-level requirement & quick start guide. For detailed
 
 ### Prerequisites
 
-* [Ruby](https://www.ruby-lang.org/en/): we recommend using [rbenv](https://github.com/rbenv/rbenv) to install the Ruby version listed on the badge.
-* [Yarn](https://yarnpkg.com/): please refer to their [installation guide](https://yarnpkg.com/en/docs/install).
-* [PostgreSQL](https://www.postgresql.org/) 9.4 or higher.
+Following software are required to be installed before you proceed.
+
+- [Ruby](https://www.ruby-lang.org/en/): we recommend using [rbenv](https://github.com/rbenv/rbenv) to install the Ruby version listed on the badge. (Alternatively, you can use [`brew`](https://brew.sh/) via `brew install ruby && brew link --overwrite ruby`).
```

Comment:
I wouldn't suggest doing `brew link --overwrite ruby`, because you overwrite the system Ruby.

Probably better to improve the documentation on using rbenv or [rvm](https://rvm.io/rvm/install)

Reply:
That's an alternative and that's what worked for me. `rbenv` version was not being picked up. It was strange. Otherwise, I would have preferred it as well.

Reply:
@ahmadawais did you put ~/.rbenv/bin and ~/.rbenv/shims in the PATH?

Reply:
I did, still didn't work.

Reply:
How about `rbenv rehash`? Does that help things?